cloud/blockstore/tools/csi_driver/internal/driver/node_test.go

```diff
@@ -663,6 +659,11 @@ func TestGetVolumeStatCapabilitiesWithoutVmMode(t *testing.T) {
 	targetFsPathPattern := filepath.Join(tempDir,
 		"pods/([a-z0-9-]+)/volumes/([a-z0-9-]+)/mount")
 
+	info, err := os.Stat(tempDir)
```

Comment:
Shouldn't we publish volume in this test instead of manually creating directory?

Reply:
It's a unit test, so I decided to skip staging/publishing and create target directory manually to reduce the test size and focus only on testing volume stat capabilities.